client/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/kubernetes-csi/external-snapshotter/client/v4
+module github.com/kubernetes-csi/external-snapshotter/client/v5
 
 go 1.15
```

Comment:
Can you also try to update this to go 1.17?  It's fine to do it in a followup PR.

Reply:
Done. Lets see if the CI passes..